sbom/sbom.mql.yaml

```diff
@@ -21,7 +22,8 @@ packs:
         mql: npm.packages { name version purl cpes.map(uri) files.map(path) }
       - uid: mondoo-sbom-kernel-installed
         filters:
-          - mql: asset.family.contains('linux')
-          - mql: asset.runtime != 'container' && asset.kind != 'container' && asset.kind != 'container-image'
+          - mql: |
```

Comment:
why do we need to split this? We can set multiple filters. Is the current approach not working?

Reply:
```
filters:
  - mql: ...
  - mql: ...
```
These filters are ORed. On Windows, this resulted in problems with the kernel query.
Windows doesn't match the first filter, but the second.

Reply:
My fault, they are or connected

Reply:
As discussed: https://github.com/mondoohq/cnspec/pull/1304